check/format-incremental

```diff
@@ -87,14 +87,23 @@ if (( only_changed == 1 )); then
 else
     echo -e "Formatting all python files." >&2
     IFS=$'\n' read -r -d '' -a format_files < \
-        <(git ls-files '*.py' ':(exclude)*_pb2.py')
+        <(git ls-files '*.py' '*.ipynb' ':(exclude)*_pb2.py')
 fi
 
 if (( ${#format_files[@]} == 0 )); then
     echo -e "\033[32mNo files to format.\033[0m"
     exit 0
 fi
 
+# Separate .py and .ipynb files
+for f in "${format_files[@]}"; do
+    if [[ "${f}" == *.ipynb ]]; then
+        format_ipynb_files+=("${f}")
+    else
+        format_py_files+=("${f}")
+    fi
+done
+
```

Comment:
Lines 110-112 - the `isort_files` array would include `.ipynb` files if they were in `format_files`, but it should only contain `.py` files.  Please adjust the condition on line 110.